widgetry/src/widgets/button.rs

```diff
@@ -220,10 +220,12 @@ impl Btn {
         let (button_geom, hitbox) = button_geom
             .batch()
             .container()
-            .padding_top(4)
-            .padding_bottom(4)
-            .padding_left(8)
-            .padding_right(8)
+            .padding(EdgeInsets {
```

Comment:
I find it nice to specify all directions of padding at once, so I introduced EdgeInsets to widgetry for padding and margins. I can revert this if it's controversial.

Reply:
Nice, it feels like widgetry is growing up into a proper UI API. :)